<ui version="4.0" >
 <class>MapViewerClass</class>
 <widget class="QWidget" name="MapViewerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>MapViewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <widget class="QToolBox" name="toolBox" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="navPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>206</width>
        <height>405</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Navigation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QRadioButton" name="MoveRadioButton" >
         <property name="text" >
          <string>Bewegung</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ZoomRadioButton" >
         <property name="text" >
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RectRadioButton" >
         <property name="text" >
          <string>Auswahlrechteck</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="InfoRadioButton" >
         <property name="text" >
          <string>Objektinformation</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>247</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="viewPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>206</width>
        <height>405</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Ansicht</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QCheckBox" name="MiniMapCheckBox" >
         <property name="text" >
          <string>Ãœbersicht</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>206</width>
        <height>405</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Bearbeiten</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <widget class="QRadioButton" name="radioButton_2" >
         <property name="text" >
          <string>Punkt</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>17</width>
           <height>302</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
